helm/templates/statefulset.yaml

```diff
@@ -1,6 +1,3 @@
-{{- if gt (.Values.replicaCount | toString | atoi) 1 }}
-{{- fail "At the moment, you can just have 1 instance of immudb. We are working to raise that limit."}}
-{{- end }}
 apiVersion: apps/v1
 kind: StatefulSet
 metadata:
```

Comment:
Again, using just one statefulset makes it impossible to distinguish between master and replicas.